float32[] lines
---
sensor_msgs/Image[] decision_paths
